lib/ramble/ramble/experimental/uploader.py

```diff
@@ -129,7 +129,11 @@ def upload_results(results):
                 logger.die('No upload URI (config:upload:uri) in config.')
 
             logger.msg(f'Uploading Results to {uri}')
-            uploader.perform_upload(uri, formatted_data)
+
+            if len(formatted_data) == 0:
+                logger.warn('No data to upload')
+            else:
+                uploader.perform_upload(uri, formatted_data)
```

Comment:
This seems like it shouldn't be part of this PR (and the rest of the PR could just be rebased if you want this change)

Reply:
Yea this got picked up accidentally, will remove (but it's actually a 0 delta from HEAD)